### PR TITLE
Updated the stub to bring it more closure to default cra template wit…

### PR DESCRIPTION
**Pull request by sahilpurav**
…h some important upgrades

### CODE REVIEW

File: template/src/setupTests.js [hunk 1]
```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom';
```
Comment (sahilpurav) [inline template/src/setupTests.js:5]: This is done as per the code present in default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/src/setupTests.js)

File: template/src/App.test.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```
Comment (sahilpurav) [inline template/src/App.test.js:6]: This is done as per the code present in default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/src/App.test.js)

File: template/src/App.js [hunk 3]
```diff
@@ -1,5 +1,3 @@
-import React from 'react';
```
Comment (sahilpurav) [inline template/src/App.js:1]: With React 17, we don't require this import.

File: template.json [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
+      "@testing-library/jest-dom": "^5.11.4",
+      "@testing-library/user-event": "^12.1.3",
+      "web-vitals": "^1.0.1"
+    },
+    "scripts": {
+      "start": "craco start",
+      "build": "craco build",
+      "test": "craco test"
+    },
+    "eslintConfig": {
```
Comment (sahilpurav) [inline template.json:17]: This is added as per the default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template.json)

**Comment by GeoffSelby** [comment]
Thanks for this. Unfortunately, I'm going to close without merging because it doesn't quite solve the problems you brought up earlier. It was however very helpful in the solution that I decided on. I will be publishing a new version, inspired by your work, shortly. Thanks again and sorry this isn't being merged.

**Pull request closed without merging**